hardhat.config.ts

```diff
@@ -128,6 +133,9 @@ const config: HardhatUserConfig = {
         artifacts: "./artifacts",
         cache: "./cache",
         sources: "./contracts",
+        deployments: './deployments',
```

Comment:
When I pull down the branch, and try to run a task (e.g. `yarn hardhat accounts`), I get an error from this line:

```
➜  pawnfi-contracts git:(hardhat-deploy) yarn hardhat accounts
yarn run v1.22.17
$ /Users/kevin/dev/pawnfi-contracts/node_modules/.bin/hardhat accounts
An unexpected error occurred:

hardhat.config.ts:136:9 - error TS2322: Type '{ artifacts: string; cache: string; sources: string; deployments: string; deploy: string; imports: string; tests: string; }' is not assignable to type 'ProjectPathsUserConfig'.
  Object literal may only specify known properties, and 'deployments' does not exist in type 'ProjectPathsUserConfig'.

136         deployments: './deployments',
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/hardhat/types/config.d.ts:168:5
    168     paths?: ProjectPathsUserConfig;
            ~~~~~
    The expected type comes from property 'paths' which is declared here on type 'HardhatUserConfig'

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Fixed after running `yarn install`